libraries/AP_HAL_Linux/Scheduler.cpp

```diff
@@ -60,6 +60,8 @@ extern const AP_HAL::HAL& hal;
         .rate = APM_LINUX_##UPPER_NAME_##_RATE,                 \
     }
 
+#define DEBUG_STACK 0
+
```

Comment:
Oops sorry. I just remember having had to remove such a debug flag from the code in the optical flow because it was not right to put it :-)